app/src/main/java/com/github/openwebnet/repository/impl/EnvironmentRepositoryImpl.java

```diff
@@ -102,4 +106,19 @@ public EnvironmentRepositoryImpl() {
         });
     }
 
+    @Override
+    public Observable<Void> delete(Integer id) {
+        return Observable.create(subscriber -> {
+            try {
+                databaseRealm.delete(LightModel.class, DomoticModel.FIELD_ENVIRONMENT_ID, id);
```

Comment:
The only thing that I really don't like is that we have to remember to modify this every time that we add a new class that extends `DomoticModel`.
Unfortunately I have no other idea now that just add a comment on the interface to remember it!